tmail-backend/tmail-third-party/openpaas/src/main/java/com/linagora/tmail/mailet/OpenPaasAmqpForwardAttribute.java

```diff
@@ -73,25 +71,31 @@
 public class OpenPaasAmqpForwardAttribute extends GenericMailet {
     private static final Logger LOGGER = LoggerFactory.getLogger(OpenPaasAmqpForwardAttribute.class);
 
-    private final Sender sender;
+    private boolean isInitialized = false;
+    private Sender sender;
+    private OpenPaasRabbitMQChannelPoolHolder openPaasRabbitMQChannelPoolHolder;
+
     private OpenPaasAmqpForwardAttributeConfig config;
 
     @Inject
-    public OpenPaasAmqpForwardAttribute(OpenPaasRabbitMQChannelPoolHolder openPaasRabbitMQChannelPoolHolder)
-        throws MailetException {
+    public OpenPaasAmqpForwardAttribute(OpenPaasRabbitMQChannelPoolHolder openPaasRabbitMQChannelPoolHolder) {
+        this.openPaasRabbitMQChannelPoolHolder = openPaasRabbitMQChannelPoolHolder;
+    }
+
+    private void initializeMailetLazly() throws MailetException {
```

Comment:
Why do we need the lazy init?

This looks like degrading production code for tests so I am rather against that move

Reply:
I'm afraid it's necessary. The alternative is to implement the second solution from https://github.com/linagora/tmail-backend/pull/1554#discussion_r1971448563, which introduces a new XML mailetcontainer construct to conditionally include a mailet.

Reply:
Fair let's move forward then